convert2rhel/utils.py

```diff
@@ -268,6 +268,16 @@ def get_traceback_str():
     """Get a traceback of an exception as a string."""
     exc_type, exc_value, exc_traceback = sys.exc_info()
     return "".join(traceback.format_exception(exc_type, exc_value, exc_traceback))
+                                              
+
+def remove_tmp_dir():
+    """Remove temporary folder (TMP_DIR), not needed post-conversion"""
```

Comment:
Nit: There is https://www.python.org/dev/peps/pep-0257/ which defines Python docstrings convention. It requires for one-line docstring to end with the dot.

Nit: Also, it might be a little better to rephrase it into something like:
"""Remove temporary directory (TMP_DIR), after successful conversion."""

Reply:
Just adding the dot here is fine. Message is good as it is since it indicates the reason why we remove the tmp dir
```suggestion
    """Remove temporary folder (TMP_DIR), not needed post-conversion."""
```

Reply:
This is Nit - so up to you guys